src/components/MAPI/clusters/guides/InspectClusterGuide.tsx

```diff
@@ -45,17 +45,21 @@ const InspectClusterGuide: React.FC<IInspectClusterGuideProps> = ({
             },
             {
               label: 'Cluster CRD schema',
-              href: docs.clusterCRDSchemaURL,
+              href: docs.crdSchemaURL(docs.crds.xk8sio.cluster),
               external: true,
             },
             {
               label: 'AzureCluster CRD schema',
-              href: docs.azureClusterCRDSchemaURL,
+              href: docs.crdSchemaURL(
+                'azureclusters.infrastructure.cluster.x-k8s.io'
```

Comment:
Could this one be grouped under `xk8sio` as well?

Reply:
Yes